cmd/manager

```diff
@@ -1 +0,0 @@
-/home/nferraro/gopaths/camel-k/src/github.com/apache/camel-k/cmd/kamel
```

Comment:
The symlink is needed for `operator-sdk generate k8s` to work. But it's broken anyway.